components/match2/wikis/arenaofvalor/match_legacy.lua

```diff
@@ -19,17 +19,21 @@ local _GAME_EXTRADATA_CONVERTER = {
 	champion = 'h',
 }
 
-function MatchLegacy.storeMatch(match2)
-	local match = MatchLegacy._convertParameters(match2)
+function MatchLegacy.storeMatch(match2, options)
+	local match = MatchLegacy.convertParameters(match2)
```

Comment:
function name doesn't match the call (one with _ prefix, one without)
same in several of the other modules too

Reply:
Good catch